server/manager/src/main/java/org/apache/accumulo/manager/tableOps/tableImport/PopulateMetadataTable.java

```diff
@@ -149,6 +153,11 @@ public Repo<Manager> call(long tid, Manager manager) throws Exception {
             if (m == null || !currentRow.equals(metadataRow)) {
 
               if (m != null) {
+                if (!sawHostingGoal) {
+                  // add a default hosting goal
+                  HostingColumnFamily.GOAL_COLUMN.put(m,
+                      new Value(TabletHostingGoal.ONDEMAND.name()));
```

Comment:
```suggestion
                      TabletHostingGoalUtil.toValue(TabletHostingGoal.ONDEMAND);
```

Just a suggestion, feel free to ignore. This adds a method call, but would make this consistent with other places in the code. For reference, this is what `TabletMutatorBase.putHostingGoal` does.